.github/workflows/runner.yml

```diff
@@ -96,15 +96,24 @@ jobs:
     - name: Checkout repository
       uses: actions/checkout@v3
 
+    - name: Install Ansible
+      run: |
+        sudo apt-get update
+        sudo apt-get -y install ansible
+
     - name: Build container
       run: |
-        docker buildx build -f ./share/containers/${{ matrix.os }}.dockerfile . --output build-out
+        pushd share/ansible/
+        ansible-playbook playbook.yml -i inventory.ini -e 'distribution=${{ matrix.os }}'
+        popd
```

Comment:
If this runs as a child script (I don't know how Ansible works), popd(1) is again unnecessary.

Reply:
This is a Github action and popd is necessary in this context